frontend/src/app/services/fullscreen.service.ts

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
New services should never go into the global `services` directory. They belong to context modules. The existing ones are just legacy ones, which never were moved. See https://github.com/DSD-DBS/capella-collab-manager/issues/320

Reply:
I wasn't sure where to place it but I'll move it to the session scope